<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexEditDialog</class>
 <widget class="QDialog" name="HexEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HexEdit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource.qrc">
    <normaloff>:/icons/images/document-binary.png</normaloff>:/icons/images/document-binary.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="4">
    <widget class="HexLineEdit" name="lineEditCodepage">
     <property name="styleSheet">
      <string notr="true">QLineEdit {border-style: outset; border-width: 1px; border-color: black}</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="HexLineEdit" name="lineEditUnicode">
     <property name="styleSheet">
      <string notr="true">QLineEdit {border-style: outset; border-width: 1px; border-color: black}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelAscii">
     <property name="text">
      <string>&amp;ASCII:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditAscii</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="HexLineEdit" name="lineEditAscii">
     <property name="styleSheet">
      <string notr="true">QLineEdit {border-style: outset; border-width: 1px; border-color: black}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelHex">
     <property name="text">
      <string>&amp;Hex:</string>
     </property>
     <property name="buddy">
      <cstring>scrollArea</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="chkEntireBlock">
     <property name="text">
      <string>&amp;Entire Block</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QScrollArea" name="scrollArea">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>361</width>
        <height>100</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelUnicode">
     <property name="text">
      <string>&amp;UNICODE:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditUnicode</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="chkKeepSize">
     <property name="text">
      <string>&amp;Keep Size</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelLastCodepage">
     <property name="text">
      <string>Last &amp;Codepage:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditCodepage</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btnCodepage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Code&amp;page...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexLineEdit</class>
   <extends>QLineEdit</extends>
   <header>HexLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditAscii</tabstop>
  <tabstop>lineEditUnicode</tabstop>
  <tabstop>btnCodepage</tabstop>
  <tabstop>lineEditCodepage</tabstop>
  <tabstop>chkKeepSize</tabstop>
  <tabstop>chkEntireBlock</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>HexEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>HexEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
